spec/requests/api/v1/users_spec.rb

```diff
@@ -87,6 +87,21 @@
         get "/api/v1/users", params: { filter: { updated_since: 1.day.ago.iso8601 } }
         expect(parsed_response["data"].size).to eql(2)
       end
+
+      context "when filtering by email" do
+        it "returns users that match" do
+          email = User.all.sample.email
```

Comment:
Out of curiosity, does `User.sample` do the same trick as `User.all.sample`?

Reply:
nah doesn't work